### PR TITLE
Initialize lerna and create single styled-system package folder

### PR DESCRIPTION
**Pull request by mfix22**
This is just the first step for initializing Lerna (part of #475), keeping supporting just the single package for now. I will update the scripts again once I create/move in the other packages 🙂

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,10 @@
...
+  "private": true,
   "scripts": {
-    "prepare": "mkdirp dist && npm run build:cjs && npm run build:esm",
-    "prettier": "prettier \"{src,test,examples}/**/*.js\" --write",
-    "clean": "rm -rf dist",
-    "build:cjs": "NODE_ENV=cjs babel src -o dist/index.cjs.js",
-    "build:esm": "NODE_ENV=esm babel src -o dist/index.esm.js",
+    "prettier": "prettier \"packages/**/{src,test,examples}/**/*.js\" --write",
+    "clean": "rm -rf packages/styled-system/dist",
+    "build": "lerna bootstrap",
```
Comment (jxnblk) [inline package.json]: This is fine as-is, but I also don't mind using `npx lerna` in cases like this

Comment (mfix22) [reply package.json]: Noted! 👍

**Comment by mfix22** [comment]
@jxnblk I have another branch ready that separates out the single package into `styled-system`, a utilities package (re: Create utilities package with themeGet, etc) and an "extras" package (name pending, re: Split style functions into separate packages #281). 

Would you like me to merge that branch into this one? 

All of these changes are not breaking ones.

**Comment by jxnblk** [comment]
@mfix22 That's awesome, thanks! I'd say you could open another PR for those changes if you like. I'd rather merge this one on its own, and I'm guessing the other PR might warrant a bit more discussion. I just want to pull it down locally and test it out before merging

**Comment by jxnblk** [comment]
Thanks for this!

**Pull request merged**